File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/W Shape3/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/W

using System;
using static System.Console;

namespace Codeforces.Assuit.Contest1
{
    class ProblemW
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            for(int i=1; i <= n; i++){
                for (int j = 0; j < (n - i); j++)
                    Write(" ");
                for (int j = 0; j < (i * 2 - 1); j++)
                    Write("*");
                WriteLine();
            }
            for(int i = n; i >= 1; i--){
                for (int j = 0; j < (n - i); j++)
                    Write(" ");
                for (int j = 0; j < (i * 2 - 1); j++)
                    Write("*");
                WriteLine();
            }
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/W Shape3/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/W
#include <iostream>
#include <cstdio>

using namespace std;

int main()
{
    unsigned short n;
    cin >> n;
    for (int i = 1; i <= n; i++)
    {
        for (int j = 0; j < (n - i); j++)
            cout << " ";
        for (int j = 0; j < (i * 2 - 1); j++)
            cout << "*";
        cout << endl;
    }
    for (int i = n; i >= 1; i--)
    {
        for (int j = 0; j < (n - i); j++)
            cout << " ";
        for (int j = 0; j < (i * 2 - 1); j++)
            cout << "*";
        cout << endl;
    }
}